src/templates/_layouts/parts-kit.html

```diff
@@ -14,14 +14,27 @@
         >
             <h2 class="parts-kit__sr-only">Parts Navigation</h2>
 
+            <label class="parts-kit__sr-only" for="pk-search">Search parts kit</label>
+            <div class="parts-kit__search">
+                <input class="parts-kit__search-input" id="pk-search" type="search"
+                    placeholder="Press / to search..."
+                    data-focus-placeholder="Type to search..."
+                    data-sidebar-search>
```

Comment:
Can you continue with the pattern and break every attribute out to a separate line?

Reply:
And then break the `>` to the last line? I think that's sort of the pattern we've adapted when you break attributes to multiple lines

```twig
<input
    class="parts-kit__search-input"
    id="pk-search"
    type="search"
    placeholder="Press / to search..."
    data-focus-placeholder="Type to search..."
    data-sidebar-search
>
```